### PR TITLE
Rule: SC2-1-2-no-keyboard-trap-standard-navigation and SC2-1-2-no-keyboard-trap-non-standard-navigation 

### PR DESCRIPTION
**Pull request by geirsf**
# Checklist for Pull Request
🚨Please review the [guidelines for contributing](https://auto-wcag.github.io/auto-wcag/pages/contribute.html) to this repository.

- [x] Make sure you are requesting to **pull a issue/feature/bugfix branch** (right side). Don't request your master branch!
- [x] Make sure you are making a pull request against the **master branch** (left side). Also you should start *your branch* off *master branch*.
- [x] Kindly prefix the pull request name with `DRAFT` or `FINAL` accordingly. For more details, read [Rule Design Guidelines](https://auto-wcag.github.io/auto-wcag/pages/design/review.html)
- [x] Provide a reference to the issue being addressed.

#Reviewing Guidelines
- For `DRAFT` PR's, leave any comments you have and approve if you’ve reviewed the entire pull request. 
- Once there have been three such reviews the pull request can be merged. 
- Any comments left will be taken up in an updated pull request, which will be necessary before a FINAL release”.
- For `FINAL` PR's, review and approve changes to be merged.

# How to Review And Approve
- Go to the “files changed” tab, there you will have the option to leave comments on different lines. 
- Once the review is completed, find the “Review changes” button in the top right, select “approve” and click “Submit review”

### Description
Please describe your pull request.

💔Thank you!

### CODE REVIEW

File: _rules/SC2-1-2-no-keyboard-trap-non-standard-navigation.md [hunk 1]
```diff
@@ -29,6 +29,8 @@ authors:
 
 The rule applies to any HTML or SVG element on a web page that is [focusable][] where focus cannot cycle to the browser UI by using [standard keyboard navigation][].
 
+**Note**: The WCAG 2.0 success criterion 2.1.2 applies to all content where focus can be moved to through keyboard navigation.
+
 ### Expectation 1
 
 For each target element help information is [visible on the page][] and [exposed to assistive technologies][] or can be accessed from within the keyboard trap.
```
Comment (WilcoFiers) [inline _rules/SC2-1-2-no-keyboard-trap-non-standard-navigation.md]: This "or can be accessed" part is a little clunky. Can this be rephrased without changing its meaning?

**Comment by WilcoFiers** [comment]
Looks great! Just one minor editorial.

**Comment by WilcoFiers** [comment]
@ShadowBB I'd appreciate a review from you as well.

**Comment by ShadowBB** [comment]
Expectation 2

"The help information explains how to cycle to the browser UI, or on how to get to a point from where it is possible to cycle to the browser UI, using standard keyboard navigation."

can be interpreted in two ways:

Interpretation 1:
1. The help information explains how to cycle to the browser UI using standard keyboard navigation.
OR
2. The help information explains how to get to a point from where it is possible to cycle to the browser UI using standard keyboard navigation.

Interpretation 2:
1. The help information explains how to cycle to the browser UI.
OR
2. The help information explains how to get to a point from where it is possible to cycle to the browser UI using standard keyboard navigation.

Please remove this ambiguity.

The rest looks great.

**Comment by jkodu** [comment]
@geirsf @DagfinnRomen - could you please address above question from @ShadowBB.

@skotkjerra @carlosapaduarte @ShadowBB - please review again.

**Comment by ShadowBB** [comment]
I will gladly review again and approve if the above ambiguity is removed. Again: the rest looks good.

**Pull request merged**